components/blobserve/pkg/blobserve/blobspace.go

```diff
@@ -216,7 +216,7 @@ func (b *diskBlobspace) AddFromTar(ctx context.Context, name string, in io.Reade
 	for _, mod := range modifications {
 		err := b.modifyFile(name, mod.Path, mod.Modifier)
 		if err != nil {
-			return xerrors.Errorf("cannot modify blob %s: %w", mod.Path, err)
+			log.WithField("path", mod.Path).WithError(err).Error("Blobspace::AddFromTar error while trying to modify file")
```

Comment:
Nit: just curious, why the change from xerrors? I ask because of [this section on error handling](https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#2c483985b5c244fc8546287f9d0fba36).

Reply:
Before it was creating a new error with `xerrors.Errorf` and returning that, what we want here is to just log if an error occurred while trying to apply the replace rule but continue the execution of the remaining rules as some of them are expected to fail because some files could not exist (vscode renamed some files but this code should work for stable version too containing old filenames)